medicai/utils/general.py

```
@@ -244,3 +245,113 @@ def interpolate_1d(input_vol, coords, axis):
+                        f"= {param.default!r}"
+                        if param.default is not inspect.Parameter.empty
+                        else ""
+                    )
+                    # annotation
+                    if param.annotation != inspect.Parameter.empty:
+                        annot_name = getattr(param.annotation, "__name__", repr(param.annotation))
+                        annot = f": {annot_name}"
+                    else:
+                        annot = ""
+                    lines.append(f"  {pname}{annot} {default}".rstrip())
+            except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)